src/ansys/simai/core/data/projects.py

```diff
@@ -30,6 +30,35 @@
     from ansys.simai.core.data.training_data import TrainingData
 
 
+class IsTrainableInfo(NamedTuple):
+    """Properties for project's trainability.
+
+    The objects of this class are associated with the output of the
+    evaluation, so that it returns a boolean variable according to
+    trainability of the project:
```

Comment:
I personally have a hard time understanding this sentence.
IMO the example + the fact that there is `def __bool__` is sufficient/more-straightforward as documentation.

Reply:
I updated the text, hope it's easier to understand it now